futaba/cogs/moderation/core.py

```diff
@@ -201,6 +211,17 @@ async def jail(self, ctx, member: MemberConv, *, reason: str = None):
 
         await self.perform_jail(ctx, member, 0, reason)
 
+    async def on_message(self, message):
+        """
+        Bot Event.
+        :param message: Messages.
+        :return: Nothing.
+        """
+        if len(message.mentions) > 5:
```

Comment:
I think this is too simplistic. In addition to the self-mentioning problem above, this also doesn't account for multiple messages in quick succession with multiple mentions.

Finally, if we have a threshold like this it should be configurable per-guild.

Reply:
Alright I might need to look further into what you do to store the configuration, also might need to somehow store a datetime of the user's last mention like this as well so that way it can pick up any and all history on that action they done?